extended example with plot and created seperate module for errorbar calculation

charnley opened the pull request.

jhjensen2 commented:
Why is output removed?
Why is statsig renamed to example?
Why are mae and me in errors.py but not used?

charnley commented:
output is added back again
errors.py is merged into example.py (and renamed back to statsig)
added me and mae graphs 
graph output is added to repo


Output was removed for the same reason as statsig was renamed for simplicity reasons, clear what the user should look at, since there was two python files. But I removed errors.py so there is only one file to run.
